### PR TITLE
PLATUI-2422: Add default content and Welsh translation for GovukTabs …

### PR DESCRIPTION
**Pull request by JoPintoPaul**
…title using Option for title

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukTabsSpec.scala [hunk 1]
```diff
@@ -17,6 +17,52 @@
...
+
+  def render(templateParams: Tabs): Try[HtmlFormat.Appendable] = {
+    implicit val request: RequestHeader = FakeRequest("GET", "/foo")
+
+    Try(component(templateParams))
+  }
+
+  "GovukTabs" when {
+    "implicit messages language is English" should {
+      "display Egnlish default title if none passed in" in {
```
Comment (kyle-bowden) [inline src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukTabsSpec.scala:41]: Small typo `Egnlish`

Comment (JoPintoPaul) [reply src/test/scala/uk/gov/hmrc/govukfrontend/views/components/GovukTabsSpec.scala]: @kyle-bowden Fixed, thank you!

File: src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html [hunk 2]
```diff
@@ -18,11 +18,11 @@
 
 @this()
 
-@(params: Tabs)
+@(params: Tabs)(implicit messages: Messages)
 @import params._
 <div@id.mapNonEmpty {id => id="@id"} class="@toClasses("govuk-tabs", classes)"@toAttributes(attributes) data-module="govuk-tabs">
   <h2 class="govuk-tabs__title">
-    @HtmlFormat.escape(title)
+    @HtmlFormat.escape({title.getOrElse(messages("govuk.tabs.titleFallbackText"))})
```
Comment (oscarduignan) [inline src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html:25]: should we `@title.map(HtmlFormat.escape _).getOrElse(messages("govuk.tabs.titleFallbackText"))`

Comment (oscarduignan) [reply src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html:25]: not sure if the syntax for that is 100% correct

Comment (JoPintoPaul) [reply src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html]: @oscarduignan Suggested change made and pushed, thank. you.

**Review by kyle-bowden — APPROVED** [review]

File: src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html [hunk 3]
```diff
@@ -18,11 +18,11 @@
 
 @this()
 
-@(params: Tabs)
+@(params: Tabs)(implicit messages: Messages)
 @import params._
 <div@id.mapNonEmpty {id => id="@id"} class="@toClasses("govuk-tabs", classes)"@toAttributes(attributes) data-module="govuk-tabs">
   <h2 class="govuk-tabs__title">
-    @HtmlFormat.escape(title)
+    @{title.map(HtmlFormat.escape _).getOrElse(messages("govuk.tabs.titleFallbackText"))}
```
Comment (oscarduignan) [inline src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukTabs.scala.html:25]: @JoPintoPaul I just realised something! The `@` will do HtmlFormat.escape for us - sorry, it's just if we want to skip escaping then we'd put `@Html(title)`

**Review by oscarduignan — APPROVED** [review]

**Pull request merged**